SIL.Windows.Forms.WritingSystems.Tests/LanguageLookupModelTests.cs

```diff
@@ -24,7 +24,7 @@ public void IncludeRegionCodes_SetToFalseSearchForChinese_ReturnsTaiwanAndMainla
 			model.IncludeRegionalDialects = false;
 			model.SearchText = "chinese";
 			string[] codes = model.MatchingLanguages.Select(li => li.LanguageTag).ToArray();
-			Assert.That(codes, Contains.Item("zh-CN"));
```

Comment:
We will need to verify that existing projects which contain this previously valid code will continue to work for them.